benchmarks/pom.xml

```diff
@@ -51,6 +51,8 @@
             <artifactId>curator-test</artifactId>
             <version>5.2.0</version>
         </dependency>
+        <!--        brauchen wir hier hier nur die api? nur -simple? nur logback?  nur provided? -->
+        <!--        https://repo1.maven.org/maven2/org/slf4j/ -->
```

Comment:
good question. I don't know. You can give it a try if you want :)

Reply:
Whoops, these logging changes were not planned to be in this PR as well ;)

Sorry for that, I guess I have to dig into this as my next todo then :)